dist/mapdc.js

```diff
@@ -26782,15 +26780,31 @@ function rasterLayerPolyMixin(_layer) {
       });
     }
 
-    var colorScaleName = getColorScaleName(layerName);
-    var scales = [{
-      name: colorScaleName,
-      type: "quantize",
-      domain: autocolors ? { data: getStatsLayerName(), fields: ["mincolor", "maxcolor"] } : state.encoding.color.domain,
-      range: colorRange,
-      nullValue: "rgba(214, 215, 214, 0.65)",
-      default: "rgba(214, 215, 214, 0.65)"
-    }];
+    var scales = [];
+    var fillColor = {};
+    var useColorScale = state.encoding.color.value === undefined;
+    if (useColorScale) {
+      var colorRange = state.encoding.color.range.map(function (c) {
+        return (0, _utilsVega.adjustOpacity)(c, state.encoding.color.opacity);
```

Comment:
I'm not sure I understand how this line works. I guess it's equivalent to `_utilsVega.adjustOpacity(c, state.encoding.color.opacity)`, but what is the purpose of the first `0`?

Reply:
Ah! @jrajav made me realize that I was reviewing the compiled file, which adds these things. So @alexbaden feel free to ignore everything that doesn't relate to an actual code change :)